plone/formwidget/autocomplete/__init__.py

```diff
@@ -1,5 +1,4 @@
 from zope.i18nmessageid import MessageFactory
-MessageFactory = MessageFactory('plone.formwidget.autocomplete')
 
-from plone.formwidget.autocomplete.widget import AutocompleteFieldWidget
-from plone.formwidget.autocomplete.widget import AutocompleteMultiFieldWidget
```

Comment:
Why did you remove this? If it was there, it means it's a part of the public API. I'm sure some package import from here instead of widget. Please revert.